simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<sdf version="1.5">
+  <model name="wamv">
+    <link name="base_link">
+      <!-- TODO: Tune these -->
+      <velocity_decay>
+        <linear>0.005</linear>
+        <angular>0.001</angular>
+      </velocity_decay>
+      <inertial>
+        <mass>250</mass>
+        <inertia>
+          <ixx>900.0</ixx>
```

Comment:
Might want to add a comment saying that this is a constant density sphere approximation.